examples/ipfs-kad/src/main.rs

```diff
@@ -58,6 +58,9 @@ async fn main() -> Result<()> {
             let store = kad::store::MemoryStore::new(key.public().to_peer_id());
             kad::Behaviour::with_config(key.public().to_peer_id(), store, cfg)
         })?
+        .with_swarm_config(|config| {
```

Comment:
```suggestion
        .with_swarm_config(|c| {
```

If you name this `c`, the formatter should but it on one line :)

Reply:
fixed 78ebe38